src/bundle/Resources/public/js/scripts/helpers/middle.ellipsis.js

```diff
@@ -1,16 +1,41 @@
 (function (global, doc, ibexa) {
-    const parseAll = () => {
+    const resizeEllipsisObserver = new ResizeObserver((entries) => {
+        entries.forEach((entry) => {
+            parseAll(entry.target);
+        });
+    });
+    const parseAll = (baseElement = doc) => {
+        if (!baseElement) {
```

Comment:
baseElement has default, so when it can be undefined?

Reply:
It won't be `undefined` but it can be `null`.